app/views/courses/ClassroomView.coffee

```diff
@@ -55,6 +54,11 @@ module.exports = class ClassroomView extends RootView
     @ownedClassrooms = new Classrooms()
     @ownedClassrooms.fetchMine({data: {project: '_id'}})
     @supermodel.trackCollection(@ownedClassrooms)
+    @levels = new Levels()
+    @levels.fetchForClassroom(classroomID, {data: {project: 'name,slug,original'}})
+    @levels.on 'add', (model) -> @_byId[model.get('original')] = model # so you can 'get' them
```

Comment:
Maybe `Levels` should have `getByOriginal`, since it seems like we mostly care about getting them by original. See: CourseDetailsView

Should this be used by line 155?
`+    level = @levels.findWhere({original: levelOriginal})`